src/main/java/com/softserveinc/ita/pageobjects/BasePage.java

```diff
@@ -0,0 +1,49 @@
+package com.softserveinc.ita.pageobjects;
+
+import static com.codeborne.selenide.Selenide.$x;
+
+public abstract class BasePage {
+
+    private String menuButtonSelector = "//button[@class = 'header__button']";
+    private String catalogButtonSelector = "//button[contains(@class, 'menu__toggle')]";
+    private String searchBarSelector = "//input[@name = 'search']";
+    private String searchButtonSelector = "//button[contains(@class, 'search-form__submit')]";
+    private String logInButtonSelector = "//rz-user[@class='header-actions__component']";
+    private String basketButtonSelector = "//rz-cart[@class = 'header-actions__component']";
+
+    public BasePage openMenu() {
```

Comment:
it's an abstract class, probably use generic and return here 'T' - that will be the class that extends this pageobject

Reply:
or it's should be here separate menu page or modal?